eclass/dist-kernel-utils.eclass

```diff
@@ -38,7 +40,19 @@ dist-kernel_build_initramfs() {
 	local rel_image_path=$(dist-kernel_get_image_path)
 	local image=${output%/*}/${rel_image_path##*/}
 
-	local args=(
+	local args=()
+	if command -v kernel-install &> /dev/null; then
```

Comment:
Tend to prefer `type -P` over `command -v` when it's bash (latter for posix), the former won't pickup functions should they exist and behaves more like calling `which` would.

Reply:
Fixed! Thanks for the review